tvtk/pyface/ui/qt4/QVTKRenderWindowInteractor.py

```diff
@@ -271,50 +177,17 @@ def _get_win_id(self):
     def Finalize(self):
```

Comment:
I tried
```
from mayavi import mlab
mlab.figure()
mlab.close()
```
with this branch on OS X 10.11.2, PySide/PyQt with Qt 4.8.7 and there is no crash.
Also run the integratedtests and no crash either.  Note that my OS X version is fairly up-to-date...